### ISSUE TITLE
Import of .env file not working

### ISSUE THREAD

**Issue by jensfranke** [prisma/vscode-graphql#60]
`.graphqlconfig` and `.env` are in the root of the project, but the Graphql Language Server failed silent. It works if the key is directly in the `.graphqlconfig`-file.

![bildschirmfoto 2018-08-11 um 17 02 19](https://user-images.githubusercontent.com/595504/43993088-ac256946-9d88-11e8-9154-e7b64b02cb89.png)

Code: 1.25.1
GraphQL: 0.0.10

**Comment by divyenduz** [comment]
Interesting, thanks for reporting. 

Can you try the `.env` file value without the quotes? 
Like: `AUTH_TOKEN_ENV=xxx`

**Comment by jensfranke** [comment]
@divyenduz Yes, it works. Thanks. But than other packages can not handle the values. I believe with quotes is the default.

**Comment by jensfranke** [comment]
hi @divyenduz, it is possible to allow also values in quotes?

**Comment by divyenduz** [comment]
Not yet I think based on this issue. I will need to investigate that.

**Comment by jensfranke** [comment]
@divyenduz thanks in advance - currently it also don't work for me without quotes.

**Comment by CoenWarmer** [comment]
Can verify that this is happening.

My `.graphqlconfig.yml`:

```
projects:
  foo:
    schemaPath: schema.graphql
    includes: ["**/*.{graphq}"]
    extensions:
      endpoints:
        default:
          url: ${env:GRAPHQL_API}
          headers:
            Authorization: "Bearer ${env:GRAPHQL_TOKEN}"
```

`graphql get-schema` downloads the schema without issue. However the VSCode plugin does not seem to validate queries. When I enter the token in the yml file directly, the plugin does work.

**Comment by peterholak** [comment]
Doesn't work on Linux either.

**Comment by divyenduz** [comment]
A fix for this is available in `0.1.8` via https://github.com/prisma/vscode-graphql/pull/98
